airbyte-integrations/connectors/source-alloydb-strict-encrypt/src/main/java/io/airbyte/integrations/source/alloydb/AlloyDbStrictEncryptSource.java

```diff
@@ -8,7 +8,7 @@
 
 import io.airbyte.integrations.base.IntegrationRunner;
 import io.airbyte.integrations.base.Source;
-import io.airbyte.integrations.source.postgres.PostgresSourceStrictEncrypt;
+import io.airbyte.integrations.source.postgres_strict_encrypt.PostgresSourceStrictEncrypt;
```

Comment:
nit: match to source postgres strict package name.
We don't usually use underscores in package names.